api/toRead.ts

```diff
@@ -0,0 +1,28 @@
+import axios from "axios"
+
+interface toReadProps {
+  id: number
+  auth: string
+}
+export const toRead = async ({ auth, id }: toReadProps) => {
```

Comment:
Since these are still books, it might be better to move this to `/books` and rename this to `getBook`